Compilers.slnf

```diff
@@ -45,20 +45,35 @@
       "src\\Compilers\\VisualBasic\\Test\\Symbol\\Microsoft.CodeAnalysis.VisualBasic.Symbol.UnitTests.vbproj",
       "src\\Compilers\\VisualBasic\\Test\\Syntax\\Microsoft.CodeAnalysis.VisualBasic.Syntax.UnitTests.vbproj",
       "src\\Compilers\\VisualBasic\\vbc\\vbc.csproj",
+      "src\\Dependencies\\CodeAnalysis.Debugging\\Microsoft.CodeAnalysis.Debugging.shproj",
```

Comment:
This is another issue that affects Mac and Linux users that we need to take into account - https://github.com/dotnet/msbuild/issues/5715

```
~/Source/roslyn [master ≡ +1 ~0 -0 !]> dotnet restore "./Compilers.slnf"
/Users/joeyrobichaud/Source/roslyn/src/Analyzers/CSharp/Analyzers/CSharpAnalyzers.shproj : Solution file error MSB5028: Solution filter file at "System.Collections.Generic.HashSet`1[System.String]" includes project "src\Analyzers\CSharp\Analyzers\CSharpAnalyzers.shproj" that is not in the solution file at "/Users/joeyrobichaud/Source/roslyn/Roslyn.sln".
```